docker/current/.env

```diff
@@ -13,3 +13,6 @@ FM_BITCOIN_RPC_URL=http://bitcoin@bitcoin@127.0.0.1:8332
 # For testing or fallback one can also use esplora
 # FM_BITCOIN_RPC_KIND=esplora
 # FM_BITCOIN_RPC_URL=https://blockstream.info/api/
+
+# fedimintd image tag from dockerhub
+FEDIMINTD_IMAGE_TAG=v0.6.0-rc.1
```

Comment:
It's probably more flexible to put a whole docker image name here, so we could test with commits too.

```
FEDIMINT_DOCKER_IMAGE=fedimint/fedimintd:v0.6.0-rc.1
```